src/components/datepicker/Datepicker.vue

```diff
@@ -12,10 +22,53 @@ export default {
     props: {
         value: Date,
         active: {
-          type: Boolean,
-          default: false
+            type: Boolean,
+            default: false
         },
-        iconPack: String
+        iconPack: String,
```

Comment:
You can remove config dependecy. As you can see Panel or Table component there is "both" prop that we'are using about custom component